### ISSUE TITLE
Detach() in route？

### ISSUE THREAD

**Issue by WhatAShot** [gram-ai/capsule-networks#36]
Why the priors.detach() is not used in the route iteration? It confused me.
